starter-code/src/index.js

```diff
@@ -1,8 +1,14 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import App from "./App.js";
+import Header from "./Header.js";
+import Section from "./section.js";
+import Nav from "./Nav.js";
 
 ReactDOM.render(
-  <App />,
+  <section>
```

Comment:
I like what you've done with components! In the index.js file, usually we only include <App />  in the render here and then link to other components from App.js. This will makes sense in the next few lessons. 

When naming components they always start with a capital letter, again you'll learn about why this is important in the next few lessons